### DISCUSSION TITLE
Custom ente 2FA icon account

### DISCUSSION THREAD

**Discussion by ui404** [ente-io/ente#3656] [Enhancements]
Hi, ente lack of icon to assign with account, is there anyway we can request icon or do custom icon? Thanks

**Comment by virtuallysurreal** [comment]
I've come across several missing services as well, and would very much like to upload my own instance to fill in those missing icons or submit them to be added. It would make it a lot easier to deal with organizing and identifying everything in a large (and ever increasing) list.
